src/west/manifest-schema.yml

```diff
@@ -66,6 +66,78 @@ mapping:
             required: true
             type: str
 
+  # The venv key specifies requirements for the build environment for Zephyr
```

Comment:
The code in this repo is supposed to be not specific to Zephyr. Is this PR Zephyr-specific? I don't have time for a full review sorry.

If it is Zephyr-specific, then it belongs to https://github.com/zephyrproject-rtos/zephyr/tree/main/scripts/west_commands

Considering `west build` is Zephyr-specific, I suspect this PR is too.

Even if it's not, https://github.com/zephyrproject-rtos/zephyr/tree/main/scripts/west_commands could be a good "staging" area to gather feedback, real-use and... more reviewers.